tests/conftest.py

```diff
@@ -150,6 +152,37 @@ async def resiliency_metrics_stack(
         yield stack
 
 
+@pytest.fixture
+def faststream_error_handler() -> FastStreamErrorHandler:
+    """Create a FastStreamErrorHandler instance.
+
+    This can be used in a test function to configure Slack error reporting for
+    a FastStream broker or router.
+    """
+    return FastStreamErrorHandler()
+
+
+@pytest_asyncio.fixture
+async def error_handling_faststream_broker(
+    faststream_error_handler: FastStreamErrorHandler,
+) -> AsyncGenerator[KafkaBroker]:
+    """Yield A FastStream Kafka broker with error reporting middleware."""
+    broker = KafkaBroker(
+        middlewares=[faststream_error_handler.make_middleware()]
+    )
+
+    @broker.subscriber("test-with-errors")
+    async def handler_with_error(foo: str) -> None:
+        _ = 1 / 0
+
+    @broker.subscriber("test-without-errors")
+    async def handler_without_error(foo: str) -> None:
+        return
+
+    async with TestKafkaBroker(broker) as br:
+        yield br
```

Comment:
Nit: This fixture looks very specific to the new tests (special routers with exception behavior, etc.), so I'd move it into the same file as the tests instead of providing it as a global fixture.